--- a/experimental/clashapi/trafficontrol/tracker.go
+++ b/experimental/clashapi/trafficontrol/tracker.go
@@ -67,6 +67,7 @@ func (t TrackerMetadata) MarshalJSON() ([]byte, error) {
 			"destinationPort": F.ToString(t.Metadata.Destination.Port),
 			"host":            domain,
 			"dnsMode":         "normal",
+			"user":            t.Metadata.User,
 			"processPath":     processPath,
 		},
 		"upload":      t.Upload.Load(),
